episodes/introduction.md

```diff
@@ -1,28 +1,213 @@
+
+You'd want to be able to **collaborate** with other people starting early on in
+the project - it'd be nice if your other team members could review your work,
+provide feedback, and contribute to parts of the analysis. Then if you have to
+disappear for a while, other people can carry the project forward.
+
+You'd want to be **open and transparent** about the data - what the inputs are,
+what the outputs are, and how you got from A to B. When you're talking to a
+potential collaborator, a new teammate, or just getting back into the project
+after a break, being able to see the core data flows can save you a lot of
+pain.
+
+You'd also want the project outcomes to be **reproducible**, so researchers in
+other labs can build on the work that you're about to do, and confirm your
+results. Maybe you end up getting a new job, leaving someone else in the lab to
+keep the project going. In your new role, you might find that you yourself want
+to build on all your old work.
+
+We'll keep these principles in mind as we work through the material in this
+course.
+
+### Welcome
+
+TODO: break the fourth wall, get students to understand that they belong here
+
+### The research life cycle
+
+Research is cyclic by nature: we ask questions about the world, and as we investigate them, we make hypotheses, test those hypotheses, and reveal more questions along with our conclusions.
+You may have seen a diagram like this in a research methods class or as part of orientation for your research institute.
+If not, that's okay!
+You don't need to know it by heart to succeed in this course.
+
+[image]
+
+KM TODO: Paragraph: summary
+
+KM TODO: Paragraph: parts of the research process which benefit from collaboration
+
+Data has its own mini-cycle. It comes into play fully at the beginning of a project, and then repeats in parts along the way.
+When you identify a new data need, either at the beginning of a project or to solve a problem when you're deep in the middle, you will need to locate an appropriate source for that data, and figure out how to access it.
+Data can be messy for many reasons, and most projects include part of the code that cleans it and prepares it for further analysis.
+Some cleaning steps might be obvious right away, and others might only be discovered after investigating odd behavior you find later.
+You might have a clear idea mathematically of what processing and analysis you want to do with the data, but perfectly translating that idea into code rarely happens on the first try.
+You will need to develop strategies for checking that your code has done the right thing, and ways to inspect the data at different points in your pipeline when something has gone wrong.
+If your project requires large quantities of data, sufficiently large that your computer cannot load it all into memory at once, you will need to use techniques and tools that are built to handle data at scale.
```

Comment:
Goal is for this paragraph to explain the diagram, and the "what we will cover" should call back to it & reinforce concepts

somehow without being too repetitive
somehow